client/utils/use-confirm-navigation.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useCallback, useRef } from '@wordpress/element';
+
+/**
+ * Hook for displaying an optional confirmation message.
```

Comment:
In this case, the message is not "optional" since we are not using WooCommerce's navigation library. But, this is not a blocker so I'm going to go ahead and approve 👍

Reply:
Good catch. I'll update the copy

Reply:
done https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2243/commits/fd9c467f18dd075342d85456e1d6d7a3e42271f9